src/main/resources/fine_tuning/models.yaml

```diff
@@ -0,0 +1,48 @@
+---
+models:
+  axiom:
+    name: axiom
+    fixed_params: "-axiom.n 30 -axiom.deterministic"
+    nexus_params:
```

Comment:
I think we could also have `nexus_params` for rm3?

Reply:
Which parameters should we be linking to?

Reply:
`rerankCutoff`？ I think it can also be used for rm3？It would save a bit time?

Reply:
oh, i see